### PR TITLE
add new versions for diligent-fx

### PR DESCRIPTION
**Pull request by AndreyMlashkin**
Specify library name and version:  **diligent/api.252009**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/README.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by AndreyMlashkin** [comment]
@prince-chrismc @SSE4 @uilianries 
please review

**Review by SSE4 — APPROVED** [review]

File: recipes/diligent-fx/all/conanfile.py [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import os
-from conans import ConanFile, tools, CMake
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conans import CMake
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, collect_libs, export_conandata_patches, apply_conandata_patches, rename
 import shutil
 
+required_conan_version = ">=1.53.0"
```
Comment (uilianries) [inline recipes/diligent-fx/all/conanfile.py:8]:
```suggestion
required_conan_version = ">=1.52.0"
```
Not using 1.53.0 features.

**Review by uilianries — COMMENTED** [review]
Please, delete line 26: `exports_sources = ["CMakeLists.txt", "BuildUtils.cmake", "script.py", "patches/**"]`

File: recipes/diligent-fx/all/conanfile.py (hunk 1 above)
Comment (AndreyMlashkin) [reply recipes/diligent-fx/all/conanfile.py]: Done, please approve

**Review by uilianries — APPROVED** [review]
LGTM

**Review by SSE4 — APPROVED** [review]

**Pull request merged**